docker-files/zeeguu-api-core/Dockerfile

```diff
@@ -24,17 +25,39 @@ RUN python setup.py install
 # https://github.com/zeeguu-ecosystem/Python-Wordstats/blob/master/wordstats/config.py#L1
 # under /opt/Zeeguu-API/wordinfo.db
 COPY --chown=www-data:www-data Zeeguu-API /opt/Zeeguu-API
+RUN rm -rf /opt/Zeeguu-API/.git
 WORKDIR /opt/Zeeguu-API
 RUN pip install -r requirements.txt
-RUN python setup.py install
+
+#ML: This is needed because the translators installer installs nltk in ~ (which in the
+# case of docker is /root. however, when run, it runs as www-data with ~ being /var/www 
+RUN python -m nltk.downloader -d /var/www/nltk_data punkt
+
+
+# comment the following lines if you do not want the FMD deployed
+# if you want it, make sure to have the config/fmd.cfg file in place 
+# pip install is here because fmd still can't be added to the 
+# requiremens... a bug prevents the API tests from running when the FMD is deployed
+
+RUN pip install flask_monitoringdashboard
+
+ENV FLASK_MONITORING_DASHBOARD_CONFIG=/opt/zeeguu/config/fmd.cfg
+COPY --chown=www-data:www-data config/fmd.cfg /opt/zeeguu/config/fmd.cfg
+ARG API_VERSION
+RUN test -n "$API_VERSION" || (echo "API_VERSION not set use --build-arg... " && false)
```

Comment:
This should also be placed at the beginning of the file, it should fail fast if you don't have the API_VERSION set for FMD.